public/components/security/roles/edit-role.tsx

```diff
@@ -50,27 +49,41 @@ export const EditRole = ({ role, closeFlyout }) => {
   async function getData() {
     try {
       setIsLoading(true);
-      const roleDataResponse = await WzRequest.apiReq('GET', '/security/roles', {
-        params: {
-          role_ids: role.id,
+      const roleDataResponse = await WzRequest.apiReq(
+        'GET',
+        '/security/roles',
+        {
+          params: {
+            role_ids: role.id,
+          },
         },
-      });
-      const roleData = (((roleDataResponse.data || {}).data || {}).affected_items || [])[0];
+      );
+      const roleData = (((roleDataResponse.data || {}).data || {})
+        .affected_items || [])[0];
       setCurrentRole(roleData);
-      const policies_request = await WzRequest.apiReq('GET', '/security/policies', {});
+      const policies_request = await WzRequest.apiReq(
+        'GET',
+        '/security/policies',
+        {},
+      );
       const selectedPoliciesCopy = [];
       const policies = (
         (((policies_request || {}).data || {}).data || {}).affected_items || []
```

Comment:
This nesting could be avoided by doing the following:
```
const affectedItems = (policies_request?.data?.data?.affected_items || []);
const policies = affectedItems.map(x => {
```

Reply:
I replaced the code to use the optional chaining operator here: https://github.com/wazuh/wazuh-kibana-app/pull/5551/commits/ff0ae1cdc18e4548d6540c1830637430b2b4e3ad